Learning about how to handle python exception error using
try: / except: / finally: